cpp/oneapi/dal/algo/kmeans/backend/gpu/empty_cluster_handling_dpc.cpp

```diff
@@ -148,6 +149,47 @@ static auto copy_candidates_from_data(sycl::queue& queue,
     return event;
 }
 
+template <typename Float>
+auto copy_candidates_from_data(sycl::queue& queue,
+                               const pr::ndview<Float, 1>& values,
+                               const pr::ndview<std::int64_t, 1>& column_indices,
+                               const pr::ndview<std::int64_t, 1>& row_offsets,
+                               const centroid_candidates<Float>& candidates,
+                               pr::ndview<Float, 2>& centroids,
+                               const bk::event_vector& deps) -> sycl::event {
+    ONEDAL_PROFILER_TASK(copy_candidates, queue);
+    ONEDAL_ASSERT(centroids.get_dimension(0) >= candidates.get_candidate_count());
+
+    const std::int64_t column_count = centroids.get_dimension(1);
+    const std::int64_t candidate_count = candidates.get_candidate_count();
+
+    const auto* values_ptr = values.get_data();
+    const auto* column_indices_ptr = column_indices.get_data();
+    const auto* row_offsets_ptr = row_offsets.get_data();
+    Float* centroids_ptr = centroids.get_mutable_data();
+    const std::int32_t* candidate_indices_ptr = candidates.get_indices().get_data();
+    const std::int32_t* empty_cluster_indices_ptr =
+        candidates.get_empty_cluster_indices().get_data();
+
+    auto event = queue.submit([&](sycl::handler& cgh) {
+        cgh.depends_on(deps);
+        cgh.parallel_for(sycl::range(candidate_count), [=](auto id) {
```

Comment:
Does it make sense to use` [=](sycl::id<1> id` or `[=](sycl::id<1> item`  instead of `[=](auto item`?